lib/spree/api/controller_setup.rb

```diff
@@ -24,7 +22,6 @@ def self.included(klass)
           prepend_view_path Rails.root + "app/views"
           append_view_path File.expand_path("../../../app/views", File.dirname(__FILE__))
 
-          self.responder = Spree::Api::Responders::AppResponder
```

Comment:
where the responders now?

Reply:
I dont think we have responders now (we dont use them for sure), I think we still get the rails default.
Anyway, in the API space, do we need them? it's only json in the api.

Reply:
what about the endpoints we haven't implemented ourselves? Don't they rely on Spree's responders?

Reply:
We are dropping spree_api altogether, all we have from now on are the endpoints we implement in OFN and all OFN api endpoints are already using AMS (that was the work done in all the previous related PRs).
New endpoints in OFN can be inspired/copied from spree_api but need to be converted to AMS. We are done with rabl :tada: